app/main/views/direct_award.py

```diff
@@ -0,0 +1,190 @@
+from flask import jsonify, abort, current_app, request
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy import asc, desc
+
+from dmapiclient.audit import AuditTypes
+from dmutils.config import convert_to_boolean
+from .. import main
+from ... import db
+from ...models import User, AuditEvent
+from app.models.direct_award import Project, Search
```

Comment:
My bad - it got put in by PyCharm's refactoring tool rather than me, but I didn't spot that :)